plugins/database/src/db_plugin.cpp

```diff
@@ -578,9 +587,7 @@ irods::error _childIsValid(
     }
     if ( status != 0 ) {
         if ( status == CAT_NO_ROWS_FOUND ) {
-            std::stringstream ss;
-            ss << "Child resource \"" << resc_name << "\" not found";
-            irods::log( LOG_NOTICE, ss.str() );
+            log_db::info("Child resource \"{}\" not found", resc_name);
```

Comment:
- Add function name prefix to log message
- `\"{}\"` -> `[{}]`

Reply:
Should all `log_db` logs that don't have a function prefix, have a function prefix?

Reply:
Yes please. We want to be able to see where the messages are coming from.

Reply:
Gotcha. Resolving this and will get in another pass.